doc/whats_new.rst

```diff
@@ -378,6 +378,11 @@ Bug fixes
    - Fix bug where stratified CV splitters did not work with
      :class:`linear_model.LassoCV`. :issue:`8973` by `Paulo Haddad <paulochf>`.
 
+   - Fix :class:`semi_supervised.LabelPropagation` to always do hard clamping.
+     Its ``alpha`` parameter now defaults to 0 and the parameter is deprecated
```

Comment:
I'm confused, the algorithm and the documentation says that `alpha` is the percentage of the initial class distribution that will be maintained. Quoting the documentation.

"The LabelPropagation algorithm performs hard clamping of input labels, which means \alpha=1."

In that case, why does this default to zero assuming the definition of alpha is the same?